hal/src/trackerm/include.mk

```diff
@@ -0,0 +1,19 @@
+
+# Define the prefix to this directory.
+# Note: The name must be unique within this build and should be
+#       based on the root of the project
+
+INCLUDE_DIRS += $(TARGET_HAL_PATH)/src/$(PLATFORM_NAME)
+# FIXME: Double check if this line is required.
+# INCLUDE_DIRS += $(HAL_MODULE_PATH)/network/ncp/wifi
```

Comment:
cc: @avtolstoy I wonder if this line needs to be uncommented since we have NCP on uhura

Reply:
You already have this below on line 16.